### ISSUE TITLE
Request for releasing a new version of cuQuantum Appliance

### ISSUE THREAD

**Issue by rht** [NVIDIA/cuQuantum#98]
[qsim v0.17.0](https://github.com/quantumlib/qsim/releases/tag/v0.17.0) has just been released, which contains a new method to run statevector simulation beyond 32 qubits (https://github.com/quantumlib/qsim/pull/623). Currently, we (BlueQubit cc: @hthayko) are blocked from using the latest cuQuantum Appliance due to this 32 qubits limitation. A new release that includes v0.17.0 would be appreciated!

**Comment by rht** [comment]
I suppose a new release will happen in conjunction with Qiskit 0.45.0? #99.

**Comment by hthayko** [comment]
@rht - Thanks for creating this request.
@leofang and @mtjrider - Do you have any estimates on when we could expect qsim v0.17 in cuQuantum?

**Comment by mtjrider** [comment]
Hi @rht @hthayko 

Thanks for your patience. To clarify, your primary request is avoiding the NumPy limitation?

**Comment by rht** [comment]
> To clarify, your primary reason for the request is avoiding the NumPy limitation?

Yes, it is due to the NumPy limitation, which [issue](https://github.com/numpy/numpy/issues/5744#issuecomment-1831830978) has been closed on NumPy main since Nov 2023.
